int64 pkg_id
int64 tot_people
P300_person[] person